src/super_context.ml

```diff
@@ -84,26 +324,39 @@ let installed_libs t = t.installed_libs
 let find_scope_by_dir  t dir  = Scope.DB.find_by_dir  t.scopes dir
 let find_scope_by_name t name = Scope.DB.find_by_name t.scopes name
 
-let expand_var_no_root t var = String.Map.find t.vars var
+let expand_vars t ~syntax_version ~var : Var.Kind.t option =
+  begin match String_with_vars.Var.destruct var with
```

Comment:
Let's add `Var.is_form : t -> bool`. This should make this code slightly clearer.